### PR TITLE
some pylint issues resolved

### PR DESCRIPTION
**Pull request by marksmayo**
All tests passed afterwards as well

### CODE REVIEW

**Comment by marksmayo** [comment]
spellchecking? frustrating as it was wrong to begin with. I'll try to fix.

**Comment by iredmail** [comment]
Is file `.vscode/settings.json` necessary? `foo.py`? etc

**Comment by marksmayo** [comment]
AH vscode is 'helpful' when you run tests. Thanks.

File: web/application.py [hunk 1]
```diff
@@ -307,7 +303,7 @@ def peep(iterator):
             return itertools.chain([firstchunk], iterator)
 
         def wsgi(env, start_resp):
```
Comment (cclauss) [inline web/application.py:305]: Let's fix this function in a separate PR because we do not support Python 2.

File: web/browser.py [hunk 2]
```diff
@@ -4,15 +4,13 @@
 import os
 import webbrowser
 from io import BytesIO
-
```
Comment (cclauss) [inline web/browser.py:7]: Please revert and do with `isort` in a separate pr.

File: web/browser.py [hunk 3]
```diff
@@ -24,7 +22,7 @@ class BrowserError(Exception):
     pass
 
 
-class Browser(object):
```
Comment (cclauss) [inline web/browser.py:27]: 👍

File: web/httpserver.py [hunk 4]
```diff
@@ -97,7 +97,7 @@ def run_wsgi_app(self):
                         return
                 except socket.timeout:
                     return
-            except:
+            except Exception:
```
Comment (cclauss) [inline web/httpserver.py:100]: 👍

File: web/session.py [hunk 5]
```diff
@@ -12,6 +12,7 @@
 from copy import deepcopy
 from hashlib import sha1
 
+from base64 import encodebytes, decodebytes
```
Comment (cclauss) [inline web/session.py:15]: isort

File: web/template.py [hunk 6]
```diff
@@ -35,7 +35,7 @@
 import tokenize
 from io import open
 import builtins
-
+from collections.abc import MutableMapping
```
Comment (cclauss) [inline web/template.py:38]: isort

**Review by cclauss — CHANGES_REQUESTED** [review]
The banisment of `else` and `elif` does not sit well with me.    Did pylint _really_ recommend this?!?

File: .vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```
[the remark this answers is not in the document]
Comment (marksmayo) [reply .vscode/settings.json]: Indeed - message R1705 - https://pheanex.github.io/pylint/

File: tests/test_application.py [hunk 8]
```diff
@@ -207,8 +207,7 @@ class blog:
             def GET(self, path):
                 if path == "foo":
                     raise web.seeother("/login", absolute=True)
-                else:
-                    raise web.seeother("/bar")
+                raise web.seeother("/bar")
```
[the remark this answers is not in the document]
Comment (marksmayo) [reply tests/test_application.py:210]: Yes - R1720 -https://vald-phoenix.github.io/pylint-errors/plerr/errors/refactoring/R1720.html

**Comment by cclauss** [comment]
Please rebase and let's see what is left.  I do not use pylint because it always requires so much configuration but other maintainers may disagree.  I would definitely disable R1705 and R1720.  I use flake8 because it finds syntax errors and undefined names and it has light touch config.

**Comment by cclauss** [comment]
We do not use pylint because we use flake8 instead.

**Pull request closed without merging**